build/gulpfile.vscode.win32.js

```diff
@@ -146,6 +147,6 @@ gulp.task(task.define('vscode-win32-ia32-inno-updater', task.series(copyInnoUpda
 gulp.task(task.define('vscode-win32-x64-inno-updater', task.series(copyInnoUpdater('x64'), updateIcon(path.join(buildPath('x64'), 'tools', 'inno_updater.exe')))));
 
 // CodeHelper.exe icon
-
+gulp.task(task.define('vscode-win32-arm64-code-helper', task.series(updateIcon(path.join(buildPath('ia32'), 'resources', 'app', 'out', 'vs', 'platform', 'files', 'node', 'watcher', 'win32', 'CodeHelper.exe')))));
```

Comment:
Don't we need an ARM64 compiled version of `CodeHelper.exe` for this?

Reply:
That would be ideal, but it currently runs fine under x86 emulation (and it's running in the background, so the performance degradation is not as noticeable).